docs/proposals/status-aggregation.md

```diff
@@ -10,39 +10,127 @@ When multiple instances of a resource (e.g. a Gateway) are running across multip
 
 ## Proposal
 
-To solve this problem, I'm proposing we leverage the status conditions in the control plane instance of that resource, aggregating the statuses to convey the necessary information.
+To solve this problem, I'm proposing we leverage the status block in the control plane instance of that resource, aggregating the statuses to convey the necessary information.
 
-For example, if the `ListenersValid` status condition type of a `Gateway` is `True` for all instances of the `Gateway` resource across all clusters, then the `Gateway` in the control plane will have the `ListenersValid` status condition type also set to `True`.
+### Status Conditions
+
```

Comment:
This first set of changes around status conditions is fixing a mistake with the example `type`. 
ListenersValid is a reason, as opposed to the type, which is `Ready`.